lib/ansible/plugins/connection/kubectl.py

```diff
@@ -23,47 +23,127 @@
+          - The namespace of the pod
+        default: ''
+        vars:
+          - name: ansible_kubectl_namespace
+        env:
+          - name: K8S_AUTH_NAMESPACE
       kubectl_extra_args:
         description:
-            - Extra arguments to pass to the kubectl command line
+          - Extra arguments to pass to the kubectl command line.
+        default: ''
+        vars:
+          - name: ansible_kubectl_extra_args
+        env:
+          - name: K8S_AUTH_EXTRA_ARGS
+      kubectl_kubeconfig:
+        description:
+          - Path to a kubectl config file. Defaults to I(~/.kube/conig)
+        default: ''
+        vars:
+          - name: ansible_kubectl_kubeconfig
+          - name: ansible_kubectl_config
+        env:
+          - name: K8S_AUTH_KUBECONFIG
+      kubectl_context:
+        description:
+          - The name of a context found in the K8s config file.
+        default: ''
+        vars:
+          - name: ansible_kubectl_context
+        env:
+          - name: k8S_AUTH_CONTEXT
+      kubectl_host:
+        description:
+          - URL for accessing the API.
+        default: ''
+        vars:
+          - name: ansible_kubectl_host
+          - name: ansible_kubectl_server
+        env:
+          - name: K8S_AUTH_HOST
+      kubectl_username:
+        description:
+          - Provide a username for authenticating with the API.
+        default: ''
+        vars:
+          - name: ansible_kubectl_username
+        env:
+          - name: K8S_AUTH_USERNAME
+      kubectl_password:
+        description:
+          - Provide a password for authenticating with the API.
+        default: ''
+        vars:
+          - name: ansible_kubectl_password
+        env:
+          - name: K8S_AUTH_PASSWORD
+      kubectl_cert_file:
+        description:
+          - Path to a certificate used to authenticate with the API.
         default: ''
         vars:
-            - name: ansible_kubectl_extra_args
-      kube_config_file:
+          - name: ansible_kubectl_cert_file
+        env:
+          - name: K8S_AUTH_CERT_FILE
+      kubectl_key_file:
         description:
-            - The path of the config file of kubectl
+          - Path to a key file used to authenticate with the API.
         default: ''
         vars:
-            - name: ansible_kube_config_file
-            - name: ansible_kubectl_config_file
-      kube_container:
+          - name: ansible_kubectl_key_file
+        env:
+          - name: K8S_AUTH_KEY_FILE
+      kubectl_ssl_ca_cert:
         description:
-            - The container to execute. If the pod has more than one container, the container name must be specified
+          - Path to a CA certificate used to authenticate with the API.
         default: ''
         vars:
-            - name: ansible_kube_container
-            - name: ansible_kubectl_container
+          - name: ansible_kubectl_cert_file
+        env:
+          - name: K8S_AUTH_SSL_CA_CERT
+      verify_ssl:
```

Comment:
How about using `kubectl_verify_ssl` instead of `verify_ssl`?

Reply:
Yep. That's a bug. Fixing.